db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Neighborhood.create(
+    [{ name: 'Chinatown' }, { name: 'Back Bay' }, { name: 'Financial District' },
```

Comment:
Line is too long. [81/80]<br>Indent the first parameter one step more than the previous line.